src/content/core/okteto-variables.mdx

```diff
@@ -203,6 +203,7 @@ Okteto provides a list of variables that serve as feature flags for some of the
 - `OKTETO_FORCE_REMOTE`:
```

Comment:
@jLopezbarb could you also remove, add, or document the rest of the feature flag envvars the CLI is using?

Reply:
Should I do it in this PR or on another one? I'll try to do it tomorrow

Reply:
Both works